libraries/WiFiClientSecure/src/WiFiClientSecure.h

```diff
@@ -80,7 +80,7 @@ class WiFiClientSecure : public WiFiClient
     void setAlpnProtocols(const char **alpn_protos);
     const mbedtls_x509_crt* getPeerCertificate() { return mbedtls_ssl_get_peer_cert(&sslclient->ssl_ctx); };
     bool getFingerprintSHA256(uint8_t sha256_result[32]) { return get_peer_fingerprint(sslclient, sha256_result); };
-    int setTimeout(uint32_t seconds);
+    int setConnectionTimeout(uint32_t milliseconds);
```

Comment:
@P-R-O-C-H-Y any clues why I still see setTimeout here?

Reply:
@me-no-dev probably the rebase went wrong. This should not be there. 
Lets open a new PR to remove it.